system/Helpers/form_helper.php

```diff
@@ -50,7 +49,7 @@ function form_open(string $action = '', $attributes = [], array $hidden = []): s
         $attributes = stringify_attributes($attributes);
 
         if (stripos($attributes, 'method=') === false) {
-            $attributes .= ' method="post"';
+            $attributes .= ' method="POST"';
```

Comment:
This seems to be `method="post"`.
https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#attr-fs-method